frontend/http/frontend.go

```diff
@@ -30,19 +30,27 @@ var promResponseDurationMilliseconds = prometheus.NewHistogramVec(
 		Help:    "The duration of time it takes to receive and write a response to an API request",
 		Buckets: prometheus.ExponentialBuckets(9.375, 2, 10),
 	},
-	[]string{"action", "error"},
+	[]string{"action", "address_family", "error"},
 )
 
 // recordResponseDuration records the duration of time to respond to a Request
 // in milliseconds .
-func recordResponseDuration(action string, err error, duration time.Duration) {
+func recordResponseDuration(action string, af bittorrent.AddressFamily, err error, duration time.Duration) {
```

Comment:
You could make this a pointer to an `AddressFamily` and pass `nil` to indicate an unknown AF.